### PR TITLE
[doc] Improve Pulsar deployment bare metal

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Security document (deploy-bare-metal section): http://pulsar.apache.org/docs/en/next/deploy-bare-metal/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
@Anonymitaet please review~ thanks

File: site2/docs/deploy-bare-metal.md [hunk 1]
```diff
@@ -30,51 +30,51 @@ Deploying a Pulsar cluster involves doing the following (in order):
...
+For machines running ZooKeeper, you had better use lighter-weight machines or VMs. Pulsar uses ZooKeeper only for periodic coordination- and configuration-related tasks, *not* for basic operations. If you run Pulsar on [Amazon Web Services](https://aws.amazon.com/) (AWS), for example, a [t2.small](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html) instance might likely suffice.
 
-#### Bookies & Brokers
+#### Bookies and Brokers
 
-For machines running a bookie and a Pulsar broker, we recommend using more powerful machines. For an AWS deployment, for example, [i3.4xlarge](https://aws.amazon.com/blogs/aws/now-available-i3-instances-for-demanding-io-intensive-applications/) instances may be appropriate. On those machines we also recommend:
+For machines running a bookie and a Pulsar broker, you had better use more powerful machines. For an AWS deployment, for example, [i3.4xlarge](https://aws.amazon.com/blogs/aws/now-available-i3-instances-for-demanding-io-intensive-applications/) instances may be appropriate. On those machines you had better use the following:
```
Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md]:
```suggestion
For machines running a bookie and a Pulsar broker, you had better use more powerful machines. For an AWS deployment, for example, [i3.4xlarge](https://aws.amazon.com/blogs/aws/now-available-i3-instances-for-demanding-io-intensive-applications/) instances may be appropriate. On those machines you can use the following:
```

File: site2/docs/deploy-bare-metal.md [hunk 2]
```diff
@@ -95,20 +95,19 @@ The untarred directory contains the following subdirectories:
...
 `conf` | Configuration files for Pulsar, including for [broker configuration](reference-configuration.md#broker), [ZooKeeper configuration](reference-configuration.md#zookeeper), and more
-`data` | The data storage directory used by ZooKeeper and BookKeeper.
-`lib` | The [JAR](https://en.wikipedia.org/wiki/JAR_(file_format)) files used by Pulsar.
-`logs` | Logs created by the installation.
+`data` | The data storage directory that ZooKeeper and BookKeeper use
+`lib` | The [JAR](https://en.wikipedia.org/wiki/JAR_(file_format)) files that Pulsar uses
+`logs` | Logs that the installation creates
 
-## Installing Builtin Connectors (optional)
+## Install Builtin Connectors (optional)
```
Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md]: This part is the same as https://pulsar.apache.org/docs/en/next/standalone/#install-builtin-connectors-optional, you can add a link here.

Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md]: This part is the same as https://pulsar.apache.org/docs/en/next/standalone/#install-builtin-connectors-optional, so you can add a link here.

File: site2/docs/deploy-bare-metal.md [hunk 3]
```diff
@@ -134,14 +133,13 @@ pulsar-io-aerospike-{{pulsar:version}}.nar
 ...
 ```
 
-## Installing Tiered Storage Offloaders (optional)
+## Install Tiered Storage Offloaders (optional)
```
Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md]: This part is the same as https://pulsar.apache.org/docs/en/next/standalone/#install-tiered-storage-offloaders-optional, so you can add a link here.

**Comment by aahmed-se** [comment]
please rebase this

**Comment by aahmed-se** [comment]
@Anonymitaet can you rebase this

**Comment by Anonymitaet** [comment]
@Monica-zy could you please resolve the conflict?

**Comment by Jennifer88huang** [comment]
run java8 tests

**Pull request merged**